h/storage.py

```diff
@@ -160,15 +154,14 @@ def create_annotation(request, data, group_service):
     return annotation
 
 
-def update_annotation(session, id_, data):
+def update_annotation(request, id_, data, group_service):
```

Comment:
Originally the `h.storage` package avoided a dependency on `pyramid.Request` but the Git history and docs don't prescribe a reason for this which still applies AFAIK.

One thing that is mentioned in the history and comments is that this module predates the introduction of services and it would make sense to convert it at some point.